src/smartpeak/include/SmartPeak/ui/Help.h

```diff
@@ -0,0 +1,95 @@
+//
+// THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+// AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+// IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+// ARE DISCLAIMED. IN NO EVENT SHALL ANY OF THE AUTHORS OR THE CONTRIBUTING
+// INSTITUTIONS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL,
+// EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO,
+// PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS;
+// OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY,
+// WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR
+// OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF
+// ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+//
+// --------------------------------------------------------------------------
+// $Maintainer: Ahmed Khalil $
+// $Authors: Ahmed Khalil $
+// --------------------------------------------------------------------------
+
+#pragma once
+#include <string>
+#include <unordered_map>
+
+namespace SmartPeak
+{
+  /**
+    @brief Documentation/Help for SmartPeakGUI
+    //TODO:auto-gen this file
+  */
+  static std::unordered_map<std::string, std::string> tooltip_info
+  {
+    // explorer windows (left)
+    {"Injections", " Use this table to view the list of injections\n and plot selected samples to the Chromatogram window"},
+    {"Transitions", " Use this table to view the list of samples\n and plot selected samples to the Features (line) window"},
+    {"Features", " Use this table to view and select from list of features\n and plot selected samples to the Features (line) window"},
+    
+    // main windows (right)
+    {"WorkflowStepWindow", " Use this window to add, edit and remove workflow steps before running the workflow"},
+    {"Calibrators", "Use this plot view calibrators in 2D"},
+    {"Features (heatmap)", "Use this window to plot features and subfeatures as a 2D heatmap"},
+    {"Features (line)", "Use this window to plot features and subfeatures as a 2D lines"},
+    {"Parameters", " Use this window to add, edit and remove parameters assocciated with the currently selected workflow"},
+    
+    {"Sequence", " Use this window to view the sequences"},
+    {"Transitions", " Use this window to view transitions"},
+    {"Spectrum", " Use this window to view the spectrum"},
+    {"Statistics", " Use this window to view statistics about feature samples and feature transitions"},
+    {"Quantitation Method", " Use this window to view quantitation methods"},
+    {"Standards Concentrations", " Use this window to view standard concentrations"},
+    {"Component Filters", " Use this window to view component filters"},
+    {"Component Group Filters", " Use this window to view component group filters"},
+    {"Component QCs", " Use this window to view component QC"},
```

Comment:
Please add `level` to the rest of these as I did above.

Reply:
I've updated all of them, PR is ready for merging.